apps/site/components/Downloads/MinorReleasesTable/index.tsx

```diff
@@ -1,10 +1,14 @@
 'use client';
```

Comment:
Does this table needs to be useclient? I don't see anything that requires it needing to be rendered on client-side.